waku/v2/protocol/waku_rln_relay/protocol_metrics.nim

```diff
@@ -8,7 +8,7 @@ import
   chronos,
   metrics,
   metrics/chronos_httpserver,
-  waku_rln_relay_constants,
```

Comment:
What are these constants related to? Static protocol? In that case, the module name should've been something likie `protocol_static_constants`

Reply:
No, these are constants generally related to rln-relay, like maximum epoch between messages, etc. This will be refactored soon.